endpoints/books/controller.py

```diff
@@ -33,38 +36,49 @@ def post(self):
         try:
             current_user = get_jwt_identity()
             book = request.get_json()
-            book['user_id'] = current_user
+            existing_book = userBookRepository.findByName(book['name'])
+            if existing_book['name'] == book['name'] and existing_book['user_id'] == current_user['id']:
+                return {'message': 'Resource already exists', 'time': datetime.datetime.now().isoformat()}, 422
```

Comment:
Just say `Book already exists`. There's no point on returning the timestamp on these requests, I think.

Reply:
Ok!